values-schema.yaml

```diff
@@ -2590,10 +2591,6 @@ properties:
                 properties:
                   bucket:
                     $ref: '#/definitions/wordCharacterPattern'
-                  project:
-                    $ref: '#/definitions/wordCharacterPattern'
-                  instance:
-                    $ref: '#/definitions/wordCharacterPattern'
```

Comment:
Should we add `serviceAccount` field (according to otomi-api loki.yaml file)?

Reply:
Yes, it should. It is added now, and also to `values-changes.yaml`.